internal/common/consts.go

```diff
@@ -48,8 +48,8 @@ const (
 	LvmDevicesPath                    = "/etc/lvm/devices/system.devices"
 	CABundleFilePath                  = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
 
-	LcaNamespace        = "openshift-lifecycle-agent"
-	Host         string = "/host"
+	EnvLCANamespace = "MY_POD_NAMESPACE"
```

Comment:
Can we convert the const to a var that's initialized via os.GetEnv, rather than calling os.GetEnv everywhere?

Reply:
I thought about it, but it is not that simple.
Can't use this `const LcaNamespace = os.getEnv("")` and I think having a global (not const) variable is not good.

Reply:
We've got a couple of cases of global vars, but ok. I'm not sure what the concern would be here. Having the global seems preferable to me, personally. Are you concerned about buggy code overwriting the var? or a store trampler somehow doing so?

Reply:
+1 also not in favour of using global variables. An alternative approach may be that we can define a common utility function to retrieve the LCA Namespace:
```golang

const DefaultLCANamespace = "openshift-lifecycle-agent"
...
func GetLCANamespace() string {
	ns := os.Getenv(common.EnvLCANamespace)
	if ns == nil {
		return DefaultLCANamespace
	}
	return ns
}
```

Reply:
dropped the commit from this PR. it was out of scope.